### DISCUSSION TITLE
learning Nuxt - whats with the app.js payload size?

### DISCUSSION THREAD

**Discussion by matt-aatechnology** [nuxt/nuxt#17586] [Questions] [answered]
Hello friends.
I've been learning vue & vuetify recently and thought i'd delve into nuxt.  I've run the nuxt build via cli and opened the browser to find the payload size of the app.js file is over 10mb - and this is before I even start building anything.

On the other hand - when I scaffold a vue app via the cli I have a total payload of about 30kb.

My inital thoughts were - I thought nuxt was supposed to be lightweight.

What am i missing here?

**Comment by TheAlexLichter** [comment, marked as answer]
Hey 👋 

Please ensure to measure when using the production build (e.g. via `yarn preview) and not during `yarn dev`.

When I build a minimal application (`npx nuxi init MY_APP`), then replacing the main template with `<h1>Hey!</h1>` and running `yarn build` and `yarn preview`, the application has a size of ~90kB (and not 10MB 😋)

Besides Vue3 and nuxt, this includes the default error page, compat logic for Nuxt2 and more.

![grafik](https://user-images.githubusercontent.com/640208/168764754-5aec4a6b-7572-47ab-91af-82f22b14911b.png)

**Comment by matt-aatechnology** [comment]
You're absolutely right thank you!

npm run generate
npm run start

I'm now seeing a payload of 96Kb.  Perfect!
